docs/understanding-radicle/faq.md

```diff
@@ -1,14 +1,135 @@
 ---
 id: faq
-title: FAQ
+title: Frequently Asked Questions
 ---
+## How do I get started?
+Head over to the [Getting Started][gs] section for instructions on how to
+download and install the Radicle Upstream client.
 
-Why can't I git clone a Radicle project?
-How is Radicle more secure than GitHub?
-How is collaborating on Radicle different than GitHub? [social model?]
-What is a remote?
-How does Radicle interact with Git?
-- https://www.git-scm.com/docs/gitremote-helpers
-What does following mean in Radicle?
-How will issues and prs work? 
-Why no CLI?
\ No newline at end of file
+## How is collaborating on Radicle different than GitHub?
+In contrast to centralized code collaboration platforms, Radicle is designed for
+bazaar-style collaboration. On the Radicle network, content is distributed
+peer-to-peer via a process called gossip. This means that peers are in control
+of their social interactions as they self-host their own content and the content
+of any peers they are interested in. This also means that within projects, there
+isn't a single `master` branch that contributors merge into. Each peer maintains
+a view of a project with their changesets and branches. These views are gossiped
+around to other peers that are interested in those changes. Read more about the
+implications and approach to this design [here][sm].
+
+## How is Radicle more secure than centralized platforms?
+The Radicle network is peer-to-peer and built on public key cryptography. To
+start, this means that there is no need to rely on third parties to access or
+use the Radicle network. It is harder to take down because there is no central
+point of failure, and is resistant to corporate capture and censorship. In
+addition, all data on the Radicle network is cryptographically signed & verified
+as it's gossiped between peers. While centralized platforms rely on user
+interface components and key oracles to signal trust from user to user, Radicle
+has designed trust into the core of the protocol.
+
+## How will issues and PRs work? 
```

Comment:
Som suggestions:
- Maybe it's worth talking about them being stored in the git object? Maybe not.
- Maybe worth mentioning it's open source and you can build your own experiences on top of it.

Reply:
Captured both. Added the second one as a question `How is Radicle licensed?`